src/modules/hooks/useBlockExplorer.test.ts

```diff
@@ -0,0 +1,53 @@
+import { renderHook } from '@testing-library/react';
+import { useChains } from 'wagmi';
+import { useBlockExplorer } from './useBlockExplorer';
+
+jest.mock('wagmi', () => ({
+    useChains: jest.fn(),
+}));
+
+describe('useBlockExplorer', () => {
+    const mockChains = [
+        {
+            id: 1,
+            name: 'Ethereum',
+            blockExplorers: { default: { url: 'https://etherscan.io' } },
+        },
+        {
+            id: 137,
+            name: 'Polygon',
+            blockExplorers: { default: { url: 'https://polygonscan.com' } },
+        },
+    ];
```

Comment:
Let's move these to the test using them, otherwise this might lead to multiple tests being coupled because using shared values

Reply:
✅ https://github.com/aragon/ods/pull/220/commits/7cea3eb11722970012e7282e7d86a963d3861f5b